packages/apps-config/src/endpoints/testingRococo.ts

```diff
@@ -139,7 +139,7 @@ export function createRococo (t: TFunction): LinkOption[] {
         // Needs paraId >= 1000, below that is reserved for system parachains
         // Update paraId to new selection, remove this flag to enable
         isDisabled: true,
-        paraId: 59,
+        paraId: 1059,
```

Comment:
Can you just drop the lines above the paraId as well - the isDisabled (and comment), with it included it won't show.

Reply:
Done